scripts/pre_commit_linter.py

```diff
@@ -463,8 +470,235 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
+def _get_all_files():
+    """This function is used to check if this script is ran from
```

Comment:
Also, I don't see such a check in the function? Would have expected to see os.getcwd somewhere

Reply:
![screenshot from 2019-03-02 04-06-20](https://user-images.githubusercontent.com/30555544/53673036-52ef4f00-3cab-11e9-80d8-5a8863b12d9f.png)
That check is being performed here, I have changed the docstring as per the function.